internal/check/timezone.go

```diff
@@ -8,7 +8,6 @@ import (
 	"github.com/hashicorp/go-cty/cty"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
-
```

Comment:
is this change needed?

Reply:
Strictly not, it was an artifact of my all my rebasing.